### PR TITLE
Added TypedArrays ES6 Sample

### LINKED ISSUE #145: ES6 TypedArray methods sample
See https://www.chromestatus.com/feature/4919908559224832

### PR DESCRIPTION
**Pull request by beaufortfrancois**
@addyosmani @jeffposnick 

With http://v8project.blogspot.de/2015/07/v8-45-release.html, I kept this (too) simple.
Hopefully, we will revisit this once https://github.com/GoogleChrome/samples/issues/167 is taking care of.

Fix #145

### CODE REVIEW

**Comment by jeffposnick** [comment]
(I don't think `BUG=...` allows you to [auto-close issues](https://github.com/blog/1506-closing-issues-via-pull-requests) via a PR, by the way.)

**Comment by beaufortfrancois** [comment]
`s/BUG/fix/` 

Thank you @jeffposnick!

**Comment by beaufortfrancois** [comment]
(gentle ping)

File: typedarray-methods-es6/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<p>
+The <code>TypedArray.of()</code> method is almost the same as <a href="https://googlechrome.github.io/samples/array-methods-es6"><code>Array.of()</code></a>.
+</p>
+{% capture array_of_js %}
+Uint8Array.of(1, 2, 3, 256 + 4); // [1, 2, 3, 4]
+{% endcapture %}
+{% include js_snippet.html js=array_of_js title='' %}
+
+<h2>Additional mirrored <code>Array.prototype</code> instance methods</h2>
+<p><code>TypedArray.prototype.copyWithin</code></p>
```
Comment (jeffposnick) [inline typedarray-methods-es6/index.html]: Can you change this to a `<ul>` with `<li>` children instead of using `<p>` to wrap each entry?

**Comment by jeffposnick** [comment]
Reviewed. Feel free to merge after addressing the one comment.

**Comment by beaufortfrancois** [comment]
Changed `<p>` to `<ul><li>`.
Thank you for reviewing.

**Pull request merged**